docs/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -206,103 +208,151 @@ mode uses block devices to create the thin pool. The following procedure shows
 
-    The command creates an LVM logical volume called `data` and an associated
-block device file at `/dev/vg-docker/data`. In a later step, you instruct the
-`devicemapper` storage driver to use this block device to store image and
-container data.
+    ```bash
+    $ lvconvert -y --zero n -c 512K --thinpool docker/thinpool --poolmetadata docker/thinpoolmeta
+    ```
 
-    If you receive a signature detection warning, make sure you are working on
-the correct devices before continuing. Signature warnings indicate that the
-device you're working on is currently in use by LVM or has been used by LVM in
-the past.
+8. Configure autoextension of thin pools via a `lvm` profile.
 
-6. Create a new logical volume (LV) called `metadata` from space in the
-`vg-docker` volume group.
+    ```bash
+    $ vi /etc/lvm/profile/docker-thinpool.profile
+    ```
 
-        $ sudo lvcreate -L 4G -n metadata vg-docker
-        Logical volume `metadata` created.
+9. Specify 'thin_pool_autoextend_threshold' value.
 
-    This creates an LVM logical volume called `metadata` and an associated
-block device file at `/dev/vg-docker/metadata`. In the next step you instruct
-the `devicemapper` storage driver to use this block device to store image and
-container metadata.
+    The value's num should be the percentage of space used before `lvm` attempts
+    to autoextend the available space (100 = disabled).
 
-7. Start the Docker daemon with the `devicemapper` storage driver and the
-`--storage-opt` flags.
+    ```
+    thin_pool_autoextend_threshold = 80
+    ```
 
-    The `data` and `metadata` devices that you pass to the `--storage-opt`
-options were created in the previous steps.
+10. Modify the `thin_pool_autoextend_percent` for when thin pool autoextension occurs.
 
-          $ sudo docker daemon --storage-driver=devicemapper --storage-opt dm.datadev=/dev/vg-docker/data --storage-opt dm.metadatadev=/dev/vg-docker/metadata &
-          [1] 2163
-          [root@ip-10-0-0-75 centos]# INFO[0000] Listening for HTTP on unix (/var/run/docker.sock)
-          INFO[0027] Option DefaultDriver: bridge
-          INFO[0027] Option DefaultNetwork: bridge
-          <output truncated>
-          INFO[0027] Daemon has completed initialization
-          INFO[0027] Docker daemon commit=1b09a95-unsupported graphdriver=aufs version=1.11.0-dev
+    The value's setting is the perentage of space to increase the thin pool (100 =
+    disabled)
 
-    It is also possible to set the `--storage-driver` and `--storage-opt` flags
- in the Docker config file and start the daemon normally using the `service` or
- `systemd` commands.
+    ```
+    thin_pool_autoextend_percent = 20
+    ```
+    
+11. Check your work, your `docker-thinpool.profile` file should appear similar to the following:
 
-8. Use the `docker info` command to verify that the daemon is using `data` and
-`metadata` devices you created.
+    ```
+     example /etc/lvm/profile/docker-thinpool.profile:
```

Comment:
The line `example /etc/lvm/profile/docker-thinpool.profile:` shouldn't be in the actual `docker-thinpool.profile` file; maybe just remove this line since it ends up being redundant.